python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -66,12 +66,23 @@ export default function Editor() {
     []
   );
 
+  const updateModel = useCallback(
+    async (_promptName?: string, _modelData?: string | ModelMetadata) => {
+      // return await ufetch.post(ROUTE_TABLE.UPDATE_MODEL,
+      //   prompt_name: promptName,
+      //   model_data: modelData,
+      // });
```

Comment:
Are you commenting this out to add it later once the `update_model()` api is built on backend?